occurrences/signals.py

```diff
@@ -1,10 +1,45 @@
+import json
+from typing import List
+
 from anymail.signals import pre_send
-from django.db.models.signals import post_delete, post_save
+from django.db import transaction
+from django.db.models.signals import m2m_changed, post_delete, post_save
 from django.dispatch import receiver
+from graphene_linked_events.utils import api_client
 from occurrences.consts import NotificationTemplate
-from occurrences.models import Enrolment
+from occurrences.models import Enrolment, Occurrence, PalvelutarjotinEvent
 from occurrences.utils import send_event_notifications_to_person
 
+from palvelutarjotin.exceptions import ApiUsageError
+
+
+def send_event_languages_update(
+    p_event: PalvelutarjotinEvent, event_language_ids: List[str]
+) -> None:
+    """
+    Update event languages to LinkedEvents.
+    """
+
+    # FIXME: This is a needless call if LE would not need a full event data.
+    # Since the LE needs at least all the required fields when an event is updated,
+    # we first need to fetch the current event object.
+    event_obj = json.loads(
```

Comment:
Good news, we already have 
`p_event.get_event_data()` for this

Reply:
I tried **HARD** using that, but I couldn't find out how I could reassign InLanguages there! The problem was that the get_event_data function creates X-objects which I found out to be readonly objects. So, I needed to call the api_client  fetcher here and restructure it differently.

Reply:
actually, after `_asdict()` call I could reassign the value, but I couldn't find out how to restructure it as basic json that LE requests, without breaking the data integrity.

Reply:
Couldn't improve. Chatted about this in slack!